### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
This doesn't seem to be backwards compatible: https://github.com/exercism/gleam-test-runner/actions/runs/3959788015/jobs/6783062011

Is it easy to install multiple version of the stdlib?

**Comment by lpil** [comment]
That's right, the exercises will need to also be updated, a branch which I'm waiting to push after this is merged.

> Is it easy to install multiple version of the stdlib?

This is not possible, no.

**Comment by ErikSchierboom** [comment]
> That's right, the exercises will need to also be updated, a branch which I'm waiting to push after this is merged.

Okay.

> We could have the test runner overwrite the exercise config to ensure that the correct versions are used. How does that sound?

Sounds like a better option than just failing. I like it.

**Comment by lpil** [comment]
Fab, I'll do that in this PR. Thanks @ErikSchierboom

**Comment by lpil** [comment]
Ready for a re-review.

File: bin/run.sh [hunk 1]
```diff
@@ -25,14 +25,23 @@ set -eu
...
 solution_dir=$(realpath "${2%/}")
 output_dir=$(realpath "${3%/}")
 results_file="${output_dir}/results.json"
 
-echo "Copying dependencies..."
+# The container environment does not have network access in order to download
+# dependencies so we copy them from a precompiled Gleam project.
+# The config for this project is also copied to ensure that the build tool does
+# not attempt to download some other version of the dependencies or any
+# additional ones.
```
Comment (ErikSchierboom) [inline bin/run.sh:37]: Excellent comment!

**Review by ErikSchierboom — COMMENTED** [review]
Lovely work! I have one suggestion: add a test case that uses the old 0.24 version, as a way to prevent regression errors.

**Comment by lpil** [comment]
@ErikSchierboom It's not easy to do that as the test runner overwrites the config in the repo itself. I could make a more sophisticated test runner which reverts the changes afterwards and write an additional script for updating the other configurations to keep the repo up-to-date, but to me this seems like not the best use of time while trying to get the track launched in a week.

**Comment by ErikSchierboom** [comment]
It is only an issue if you then check in the changed file. I see your argument, but the backwards compatibility is something you'd want to test. But I'm happy for this to be merged.

**Comment by ErikSchierboom** [comment]
If you give me 10 mins, I'll push an update

**Comment by ErikSchierboom** [comment]
It'll be a little longer. Feel free to merge

**Comment by ErikSchierboom** [comment]
@lpil I've pushed two commits:

- https://github.com/exercism/gleam-test-runner/pull/9/commits/bff727d92d146f501194e64d64d61b21375eb912: adds test test case
- https://github.com/exercism/gleam-test-runner/pull/9/commits/d2a04ac4934853443fc50933c3fcb591cfaab432: restore the files

**Comment by ErikSchierboom** [comment]
I think this is ready to merge!

**Pull request merged**